### PR TITLE
Use util.inspect on objects being logged.

### PR DESCRIPTION
**Pull request by Half-Shot**
This came about because we used to use ``console.log``, which did this implicitly but winston does not. I think this is a fairly sane suggestion as long as we don't try to rapidly log lots of objects, though I don't think ``util.inspect`` has any extreme performance penalties.

### CODE REVIEW

File: lib/Logging.js [hunk 1]
```diff
@@ -124,7 +122,7 @@ class LogWrapper {
     _log(messageParts, type) {
         messageParts = this._formatParts(messageParts);
         if(this.logger == null) {
-            this.messages.push([{type, messageParts}]);
+            this.messages.push({type, messageParts});
```
Comment (lukebarnard1) [inline lib/Logging.js]: This looks unrelated

File: lib/Logging.js [hunk 2]
```diff
@@ -65,7 +63,7 @@ class Logging {
     }
 
     Get(name) {
-        if(!this.loggers.has(name)) {
+        if (!this.loggers.has(name)) {
```
Comment (lukebarnard1) [inline lib/Logging.js]: This likewise looks unrelated.

File: lib/Logging.js [hunk 3]
```diff
@@ -39,24 +39,22 @@ class Logging {
...
-            this.transports.push(new (winston.transports.DailyRotateFile)({
-                filename,
-                datePattern: "YYYY-MM-DD",
-                name: `logfile` + i,
-                formatter: formatterFn,
-                level,
-                timestamp: timestampFn,
-                maxFiles: config.maxFiles > 0 ? config.maxFiles : undefined
-            }));
+        if(config.files !== undefined) {
```
Comment (lukebarnard1) [inline lib/Logging.js]: Is this actually a functional change? It doesn't seem like it.

File: lib/Logging.js (hunk 1 above)
Comment (lukebarnard1) [reply lib/Logging.js]: *To the commit

File: lib/Logging.js (hunk 2 above)
Comment (lukebarnard1) [reply lib/Logging.js]: *to the commit

File: lib/Logging.js (hunk 1 above)
Comment (Half-Shot) [reply lib/Logging.js]: Yep, but it turned out this was doing the wrong thing and I felt it wasn't worth creating a PR just to fix a bug.

File: lib/Logging.js (hunk 3 above)
Comment (Half-Shot) [reply lib/Logging.js]: It's because of the old code:
```
if(config.files === undefined) {
            return;
}
```
which would cause  ``wrapper.setLogger`` to not be called if ``config.files`` wasn't defined. So a console only logging config would just not work.

File: lib/Logging.js (hunk 1 above)
Comment (lukebarnard1) [reply lib/Logging.js]: At the least it should be part of a different commit, PR/branch history is less important.

File: lib/Logging.js (hunk 3 above)
Comment (lukebarnard1) [reply lib/Logging.js]: Oh, I missed line 59 onwards, ignore me.

File: lib/Logging.js (hunk 2 above)
Comment (Half-Shot) [reply lib/Logging.js]: This was just styling fixes..although it looks like I don't keep to a style anyway

**Comment by Half-Shot** [comment]
Note to self: Setup linting.

**Comment by Half-Shot** [comment]
Split commits up, did a styling fix too so we are more consistent.

**Comment by Half-Shot** [comment]
Thank you ^^

**Comment by lukebarnard1** [comment]
>Thank you ^^

Most welcome, thanks for splitting things :+1:
